pkg/metrics/constants.go

```diff
@@ -26,6 +26,12 @@ const (
 
 	ErrorLabel       = "error"
 	ProvisionerLabel = "provisioner"
+
+	// Reasons for CREATE/DELETE shared metrics
+	ConsolidationReason = "consolidation"
+	SchedulingReason    = "scheduling"
```

Comment:
Thoughts?
```suggestion
	ProvisioningReason    = "provisioning"
```

Reply:
Sure, I went back and forth on this one anyways 😄